CAM/AttachCamToPedBone.md

```diff
@@ -12,7 +12,7 @@ void ATTACH_CAM_TO_PED_BONE(Cam cam, Ped ped, int boneIndex, float xOffset, floa
 ## Parameters
 * **cam**: 
 * **ped**: 
-* **boneIndex**: 
+* **boneId**: 
```

Comment:
Hi, just reminding you that you forgot to change `boneIndex` to `boneId` on the native parameter located on line 8 as AvarianKnight mentioned. Both (parameters and function declaration) must match for the checks to pass.